zuul-core/src/main/java/com/netflix/zuul/filters/endpoint/ProxyEndpoint.java

```diff
@@ -365,8 +367,15 @@ public HttpResponseMessage apply(HttpRequestMessage input) {
     @Override
     public HttpContent processContentChunk(ZuulMessage zuulReq, HttpContent chunk) {
         if (originConn != null) {
+            if (chunk instanceof LastHttpContent && !receivedChunkAfterProxyStarted) {
```

Comment:
this is a subtly change. If a last content check comes in after a connection has been established (e.g. a pooled connection is used), and no other content chunks have arrived in this state, then buffer the last http content so requests can be eligible for retries